src/layers/tile/finite.rs

```diff
@@ -7,6 +7,7 @@ use crate::{
 
 use super::util::parse_data_line;
 
+/// The raw data of a [`FiniteTileLayer`]. Does not include a reference to its parent [`Map`](crate::Map).
```

Comment:
Same here, the section about the parent is probably only relevant for `TileLayerData`.

Reply:
It's relevant to all map wrapped objects; they all have the same docs but with different type references

Reply:
Ah, right.